fares-data-build-tool/create-data
# feat: cfd-481 - fix update fare triangle

# Description

-   Implement the missed acceptance criteria and refactored code.

# Testing instructions

-   From `/products/productDetails` page, attempt to edit fare triangle and edit the CSV to ensure error handling is working (e.g. edit csv by adding new fare stage, and also test by changing the name on one of the fare stages but keep the same number of stages)

# Type of change
